### DISCUSSION TITLE
How can I support https://embed.diagrams.net function in my offline network？

### DISCUSSION THREAD

**Discussion by wucy-personal** [jgraph/drawio#2553] [Q&A]
I want to use https://embed.diagrams.net to embed mode for my wordpress website
But https://embed.diagrams.net is online support, and my wordpress website is offline
I try to use offline drawio, but it not support embed mode functions
-----------------------------
https://embed.diagrams.net 网站支持交互式的离线嵌入功能，还是必须联网访问，离线场景下如何使用？私有化的部署draw.io能够开启这个模式功能？

**Comment by Peeeet** [comment]
I would highly recommend you use the "export as SVG" function of diagrams.net offline to use in WordPress.  You do need to enable SVG uploads in WordPress (blocked by default) but there are free plugins in the WordPress Plugin Directory to do this as long as the public does not have access to your WordPress site (where they could upload malicious SVG files).

If you need to edit the diagrams as well, consider using the Offline downloaded software and then saving the diagram as an SVG with the embedded diagram file in it so that you can come back and make changes with diagrams.net software and then re-export as SVG again.  It's not as fluid as being able to edit the diagram in-line, but SVG export gives you all the benefits of a vector based file format along with the editing abilities of Diagrams.net

**Reply by wucy-personal** [reply]
Thanks，"export as SVG"  is not worked well in Wordpress，I use"export as PNG"  instead of~~

Wordpress plugins(  https://wordpress.org/plugins/drawit/ ) need a online site, is best to deploy https://embed.diagrams.net offline~
